src/Administration/Resources/app/administration/src/module/sw-settings-measurement/snippet/de-DE.json

```
@@ -0,0 +1,23 @@
+{
+  "sw-settings-measurement": {
+    "general": {
+      "mainMenuItemGeneral": "Maßeinheitensystem",
+      "description": "Modul zum Ändern der Standard-Maßeinheiten.",
+      "textHeadline": "Maßeinheitensystem"
+    },
+    "defaultUnits": {
+      "title": "Standard-Einheiten",
+      "description": "Lege fest, welche Maßeinheiten in der Administration standardmäßig verwendet werden. Welche Einheiten Deine Kunden sehen, bestimmst Du in den {link}.",
```

The description could be a little misleading here. By saying it is used in the Administration we imply, that this is being used in the whole administration, which is not really true. For example, it does not have an effect on which unit is selected by default on the product detail page. It only defines the defaul selection of units in the sales channel config.

```suggestion
      "description": "Lege fest, welche Maßeinheiten standardmäßig in neuen Verkaufskanälen verwendet werden. Welche Einheiten Deine Kunden sehen, bestimmst Du in den {link}.",
```

---

@ralfhoefer would you agree?

---

Good point, we should do it your way 👍.